src/aerie_cli/persistent.py

```diff
@@ -21,15 +21,53 @@
     APP_DIRS.user_config_dir).resolve().absolute()
 CONFIGURATION_FILE_PATH = CONFIGURATION_FILE_DIRECTORY.joinpath('config.json')
 
-SESSION_FILE_DIRECTORY = Path(APP_DIRS.user_config_dir).resolve().absolute()
+SESSION_FILE_DIRECTORY = Path(APP_DIRS.user_config_dir).resolve().absolute() / "session_files"
+SESSION_FILE_DIRECTORY.mkdir(parents=True, exist_ok=True)
 SESSION_TIMESTAMP_FSTRING = r'%Y-%jT%H-%M-%S.%f'
 SESSION_TIMEOUT = timedelta(hours=12)
 
+MAX_LOG_FILES = 25
+
+TIME_FORMAT = '%Y-%m-%d_%H:%M:%S'
+START_TIME = datetime.now().strftime(TIME_FORMAT)
+LOGS_PATH = Path(APP_DIRS.user_config_dir).resolve().absolute() / "logs"
+LOGS_PATH.mkdir(parents=True, exist_ok=True)
+CURRENT_LOG_FILE_NAME = f"aerie_cli_{START_TIME}"
+CURRENT_LOG_PATH = LOGS_PATH / (f"{CURRENT_LOG_FILE_NAME}.log")
+number_appended_to_log = 1
+while CURRENT_LOG_PATH.exists():
+    CURRENT_LOG_PATH = LOGS_PATH / (f"{CURRENT_LOG_FILE_NAME}_{number_appended_to_log}.log")
```

Comment:
This feels like values and logic that would be more cleanly tracked in a `PersistentLogs` class. For the while loop specifically, we could also solve this by appending a [UUID](https://docs.python.org/3/library/uuid.html#uuid.uuid4), which will avoid the race condition where two Aerie-CLI processes both think they've found an available log path before creating it.